redirects.yaml

```diff
@@ -53,6 +53,7 @@ aboutus/online-accounts-legal-notice/?: "/legal/online-account-terms"
 aboutus/privacypolicy/?: "/legal/data-privacy"
 aboutus/privacypolicy/thirdparties/?: "/legal/data-privacy"
 aboutus/trademarkpolicy.*/?: "/legal/intellectual-property-policy"
+activedirectory/?: "/server/docs/service-sssd"
```

Comment:
the reason why I moved it too `/desktop/activedirectory` instead of a top level domain was mostly so we don't get used to this pattern of reserving toplevel domains.

If you think this won't be an issue and that we have hard requirements I don't mind putting this at this level.

Reply:
There are two issues with including `/desktop`:
- We’re displaying the URL in an installer error dialog, for you to write down / photograph and use later (because visiting it now is premature). So the shorter, the better.
 - The redirect goes (both before and after this change) to a server-focused page, not a desktop one.

However, I just realised that in the two other cases where the installer needs to do this, the URL is help.ubuntu.com/topic, not ubuntu.com/topic. So I guess it would make more sense to put this redirect on help.ubuntu.com instead.

Reply:
@jibel has kindly updated the MP for the dialog to display “help.ubuntu.com/activedirectory” instead. So the ubuntu.com redirect can be dropped altogether. Sorry for taking your time on this.